content/blog/tidymodels-2022-q2/index.Rmd

```diff
@@ -0,0 +1,206 @@
+  "plsmod", "poissonreg", "rules", "recipes", "embed", "themis", "textrecipes",
+  "workflows", "workflowsets", "tune", "yardstick", "broom", "dials", "butcher",
+  "hardhat", "infer", "stacks", "tidyposterior", "tidypredict"
+))
+```
+
+- [baguette](https://baguette.tidymodels.org/news/index.html#baguette-100)
+- [broom](https://broom.tidymodels.org/news/index.html#broom-080)
+- [butcher](https://butcher.tidymodels.org/news/index.html#butcher-020)
+- [dials](https://dials.tidymodels.org/news/index.html#dials-100)
+- [discrim](https://discrim.tidymodels.org/news/index.html#discrim-100)
+- [embed](TODO)
+- [hardhat](https://hardhat.tidymodels.org/news/index.html#hardhat-110)
+- [infer](https://infer.tidymodels.org/news/index.html#infer-v102)
+- [modeldata](TODO)
+- [multilevelmod](https://multilevelmod.tidymodels.org/news/index.html#multilevelmod-100)
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-100)
+- [plsmod](TODO)
+- [poissonreg](https://poissonreg.tidymodels.org/news/index.html#poissonreg-100)
+- [recipes](TODO)
+- [rsample](https://rsample.tidymodels.org/news/index.html#rsample-100)
+- [rules](https://rules.tidymodels.org/news/index.html#rules-100)
+- [spatialsample](https://spatialsample.tidymodels.org/news/index.html#spatialsample-020)
+- [stacks](https://stacks.tidymodels.org/news/index.html#stacks-023)
+- [textrecipes](TODO)
+- [themis](TODO)
+- [tidyposterior](https://tidyposterior.tidymodels.org/news/index.html#tidyposterior-100)
+- [tidypredict](https://tidypredict.tidymodels.org/news/index.html#tidypredict-049)
+- [tune](TODO)
+- [workflows](TODO)
+- [workflowsets](TODO)
+- [yardstick](https://yardstick.tidymodels.org/news/index.html#yardstick-100)
+
+The `NEWS` files are linked here for each package; you'll notice that there are a lot! We know it may be bothersome to keep up with all these changes, so we want to draw your attention to our recent blog posts above and also highlight a few more useful updates in today's blog post.
+
+## case weights
+
+Much of the work we have been doing so far this year has been related to case weights. For a more detailed account of the deliberations see this earlier post about the [use of case weights with tidymodels](https://www.tidyverse.org/blog/2022/05/case-weights/).
+We are confident that we have created a good foundation with our implementation and we are signaling the support for case weights by bumping the version of affected packages and extension packages to 1.0.0.
+
+A full worked example can be found in the  [prevous blog post](tidyverse.org/blog/2022/05/case-weights/#tidymodels-syntax) and on [the tidymodels site](TODO).
+
+## non-standard roles in recipes
+
+Recipes use the idea of roles to determine how and when the different variables are used. The main roles you have seen are "outcome", "predictor", and now "case_weights". You are also able to change the roles of these variables using `add_role()` and `update_role()`.
```

Comment:
```suggestion
recipes use the idea of roles to determine how and when the different variables are used. The main roles you have seen are "outcome", "predictor", and now "case_weights". You are also able to change the roles of these variables using `add_role()` and `update_role()`.
```
Do we capitalize because it's the beginning of a sentence or do we keep it lower case because that's its name?

Reply:
The style guide says to not capitalize but instead prefer "The recipes package". Alternatively, we could treat it like a noun, i.e. "create a recipe", which would also be capitalized.

Reply:
Let's change "you have seen" since it's not in this post itself and it would be good for this post to not make super strong assumptions about what folks know already.